packages/liquidator/test/Liquidator.js

```diff
@@ -1272,185 +1272,10 @@ contract("Liquidator.js", function(accounts) {
             assert.ok(liquidator);
           });
           versionedIt([{ contractType: "any", contractVersion: "any" }])(
-            "full liquidation: should enable and not affect existing logic if not triggered",
```

Comment:
im good with removing tests. just would like your comment on why this is possible now

Reply:
This test checks that the liquidator will use its "full balance" if the `whaleDefnseFundWei` is set to a trivially small value:

```
in this test, the WDF is not triggered because the liquidator has enough available capital
to liquidate a full position.
```

So this case is not relevant anymore